### PR TITLE
Add beta link support

### PR DESCRIPTION
**Pull request by PanSpagetka**
The beta badge will be green once https://github.com/RedHatInsights/insights-chrome/pull/1306 is merged. Also Roles are not really beta feature, I just make it look like it is for the sake of the screenshot.

Screenshot from 2021-04-01 09-35-31

Screenshot from 2021-04-06 11-23-46

https://issues.redhat.com/browse/RHCLOUD-13180
@mmenestr

### CODE REVIEW

File: src/js/App/Sidenav/Navigation.scss [hunk 1]
```diff
@@ -2,3 +2,21 @@
...
+    text-align: center;
+    width: 90%;
+    div {
+        margin: var(--pf-global--spacer--md);
+    }
+    .info-icon {
+        color: var(--pf-global--info-color--100);
+        margin: var(--pf-global--spacer--md);
+    }
+}
```
Comment (fhlavac) [inline src/js/App/Sidenav/Navigation.scss:22]: Add new line

**Review by fhlavac — COMMENTED** [review]
Maybe add some spacing between the skeletons and badge when loading?

Screenshot from 2021-04-01 10-09-27

**Review by fhlavac — COMMENTED** [review]
Tested it locally and except for a few details listed below, it looks good to me :-)

File: src/js/App/Sidenav/Navigation.scss [hunk 2]
```diff
@@ -2,3 +2,21 @@
     align-items: center;
     svg { margin-left: var(--pf-global--spacer--sm); }
 }
+
+.ins-c-navigation__beta-badge {
+    float: right;
+    margin-left: auto;
+    margin-right: 0px !important;
```
[the remark this answers is not in the document]
Comment (karelhala) [reply src/js/App/Sidenav/Navigation.scss:9]: I'd hide the `beta` tags when nav is loading. It looks kinda strange.

File: src/js/App/Sidenav/BetaInfoModal.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { Title } from '@patternfly/react-core/dist/js/components/Title/Title';
+import { Modal, ModalVariant } from '@patternfly/react-core/dist/js/components/Modal/Modal';
+import { Button } from '@patternfly/react-core/dist/js/components/Button/Button';
+import { Bullseye } from '@patternfly/react-core/dist/js/layouts/Bullseye/Bullseye';
+import './Navigation.scss';
+
+import PropTypes from 'prop-types';
+
+const BetaInfoModal = ({ isOpen, onClick, onCancel, menuItemClicked }) => {
+  return (
```
Comment (karelhala) [inline src/js/App/Sidenav/BetaInfoModal.js:12]: Extra `return` statement.

File: src/js/App/Sidenav/ExpandableNav.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ const ExpandableNav = ({ subItems, onClick, title, id, active, ignoreCase, activ
             isActive={active && subItem.id === activeApp}
             onClick={(event) => onClick(event, subItem)}
             navigate={subItem?.navigate}
+            isPublicBeta={subItem?.isPublicBeta}
```
Comment (karelhala) [inline src/js/App/Sidenav/ExpandableNav.js:22]: I wouldn't name it `isPublicBeta`, but rather `isBeta` as there is already this field used in the config.

File: src/js/App/Sidenav/__snapshots__/SectionNav.test.js.snap [hunk 5]
```diff
@@ -52,13 +52,13 @@ exports[`SectionNav should render app with subapp and custom url, but without gr
           data-ouia-component-type="PF4/NavItem"
           data-ouia-safe="true"
         >
-          <a
+          <div
```
Comment (karelhala) [inline src/js/App/Sidenav/__snapshots__/SectionNav.test.js.snap:55]: It's really strange that all `a` elements were replaced with `div`

File: src/js/App/Sidenav/ExpandableNav.js (hunk 4 above)
Comment (ryelo) [reply src/js/App/Sidenav/ExpandableNav.js]: Agreed with this, but we should also go through and remove things that are not actually in beta anymore

Comment (ryelo) [reply src/js/App/Sidenav/ExpandableNav.js]: https://github.com/RedHatInsights/cloud-services-config/pull/488

File: src/js/App/Sidenav/__snapshots__/SectionNav.test.js.snap (hunk 5 above)
Comment (ryelo) [reply src/js/App/Sidenav/__snapshots__/SectionNav.test.js.snap]: Yeah this should be double checked. There should still be a `a` tag in here

File: src/js/App/Sidenav/ExpandableNav.js (hunk 4 above)
Comment (ryelo) [reply src/js/App/Sidenav/ExpandableNav.js]: @PanSpagetka The `isBeta` in the API field is strictly for the API. I've added a field for `isBeta` in the frontend too. We should check on that one. It's in the PR above

**Comment by PanSpagetka** [comment]
@karelhala @fhlavac I think all issues should be resolved by now

**Review by fhlavac — APPROVED** [review]

**Comment by mmenestr** [comment]
The only comment I have is the spacing between the body and the buttons should be 32px! Other than that, looks great!
(sorry I didn't see this earlier)

**Review by karelhala — APPROVED** [review]

**Comment by PanSpagetka** [comment]
I have changed spacing between body and buttons to 32px and updated screenshot.

**Pull request merged**